client/ayon_core/plugins/publish/extract_otio_review.py

```diff
@@ -520,9 +526,12 @@ def _render_segment(self, sequence=None,
             input_extension
             and self.output_ext == input_extension
         ):
-            command.extend([
-                "-c", "copy"
-            ])
+            if input_extension.lower() in [
```

Comment:
Does this condition actually make sense? This happens only if input extension is same as `output_ext`. And `output_ext` is always `".png"`, so why do we need this condition?